patches/minecraft/net/minecraft/world/level/chunk/LevelChunk.java.patch

```diff
@@ -5,18 +5,10 @@
  import org.slf4j.Logger;
  
 -public class LevelChunk extends ChunkAccess {
-+public class LevelChunk extends ChunkAccess implements net.minecraftforge.common.capabilities.ICapabilityProviderImpl<LevelChunk> {
++public class LevelChunk extends ChunkAccess implements net.minecraftforge.common.attachment.IAttachmentHolder {
```

Comment:
Based on this, attachments seem to be only possible on `LevelChunk`s (which is consistent with capabilities) - is it able to be extended to `ProtoChunk`s or is that out of scope / future work / not intended? That would be very useful for saving data to chunks between world generation stages as my current workaround involves having to track proto chunks explicitly and manage save/load independently.

Reply:
I think it's in scope for future work. This PR is already massive, and I didn't want to add more to it.